<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск Эйлерова пути</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QFrame" name="frGraphics">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="grbGraph">
          <property name="title">
           <string>Параметры графа</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Количество вершин</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spbVCount">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Заполненность ребер (%)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spbEProcent">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnGenerate">
             <property name="text">
              <string>Сгенирировать автоматически</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnInput">
             <property name="text">
              <string>Ввести вручную</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grbVisualization">
          <property name="title">
           <string>Параметры визуализации</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Задержка (сек)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spbDelay">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>60</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnReset">
             <property name="text">
              <string>Вернуться в начало</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStartAuto">
             <property name="text">
              <string>Запустить автоматический режим</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnShowResult">
             <property name="text">
              <string>Перейти к результату</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Номер шага:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblStep">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btnStepBack">
               <property name="text">
                <string>&lt;&lt; Шаг назад</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnStepFront">
               <property name="text">
                <string>Шаг вперед &gt;&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
